src/nvim/api/buffer.c

```diff
@@ -1134,9 +1134,63 @@ Boolean nvim_buf_is_valid(Buffer buffer)
   return ret;
 }
 
+/// Deletes a named mark in the buffer
+//
+/// Marks are (1,0)-indexed. |api-indexing|
+///
+/// @note Only deletes marks associated with the buffer
```

Comment:
i would cram this into the first line, it's surprising behavior. Especially that it silently ignores them.

Probably we should throw an error instead.